doc/modules/decomposition.rst

```diff
@@ -172,27 +172,6 @@ Note: the implementation of ``inverse_transform`` in :class:`PCA` with
       Halko, et al., 2009
 
 
-.. _kernel_PCA:
```

Comment:
Any specific reason to move this below?

Reply:
Yes I think that kernel PCA deserves higher visibility, in other words it should appear in the toc that is generated on the left of the page (as of now you see only PCA). It makes navigation a bit clearer to separate all things relative to PCA, from all things related to kPCA

Reply:
For an easier review:

 - Current doc: https://scikit-learn.org/stable/modules/decomposition.html (no kPCA in the toc on the left, this is in an obscure subsection)
 - New proposal: kPCA is easy to spot https://100494-843222-gh.circle-artifacts.com/0/doc/modules/decomposition.html